apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/McsClaimResponseTransformerV2.java

```diff
@@ -132,21 +141,37 @@ private static ClaimResponse.ClaimResponseStatus getStatus(RdaMcsClaim claimGrou
     return status;
   }
 
+  /**
+   * Gets the associated {@link ClaimResponse.RemittanceOutcome} for the given {@link RdaMcsClaim}
+   * object's status code.
+   *
+   * @param claimGroup The {@link RdaMcsClaim} object to get the status from.
+   * @return The {@link ClaimResponse.RemittanceOutcome} associated with the given data's status
+   *     code.
+   */
   private static ClaimResponse.RemittanceOutcome getOutcome(RdaMcsClaim claimGroup) {
     ClaimResponse.RemittanceOutcome outcome;
 
     if (claimGroup.getIdrStatusCode() == null) {
-      outcome = ClaimResponse.RemittanceOutcome.QUEUED;
+      outcome = ClaimResponse.RemittanceOutcome.PARTIAL;
```

Comment:
I think we may need to revert this change.  For both FISS and MCS, I'd like the empty string or NULL values to map to QUEUED.  Its the other random characters that we might see, like `&` or `+`, actually character values that aren't defined in our mappings, should be mapped to PARTIAL.  I'll update the confluence page with this.